dev-libs/libphonenumber/libphonenumber-8.12.26.ebuild

```diff
@@ -17,16 +17,21 @@ LICENSE="Apache-2.0"
 SLOT="0"
 
 RDEPEND="
-	dev-libs/boost
-	dev-libs/icu
-	dev-libs/protobuf
+	dev-libs/boost:=
+	dev-libs/icu:=
+	dev-libs/protobuf:=
+	>=virtual/jre-1.8:*
 "
-
-BDEPEND="
-	dev-cpp/gtest
-	virtual/jdk
+DEPEND="
+	${RDEPEND}
+	>=virtual/jdk-1.8:*
 "
+BDEPEND="dev-cpp/gtest"
```

Comment:
If cmake files have unconditional references to gtest, then you can add patch to remove them.
But even without it, valid place for `dev-cpp/gtest` would be `DEPEND`, not `BDEPEND`.

Reply:
CMakeFiles are messed up with gtest and It's a bit difficult to patch them. I'm trying to take it out

Reply:
@Arfrever anyway why DEPEND instead of BDEPEND? It's required only during build time